ui/analyse/src/plugins/nvui.ts

```diff
@@ -247,6 +248,17 @@ export default function (redraw: Redraw) {
   };
 }
 
+function renderCurrentLine(ctrl: AnalyseController, style: Style): (string | VNode)[] {
+  if (ctrl.path.length === 0) {
+    return renderMainline(ctrl.mainline, ctrl.path, style);
+  } else {
+    const currentLine = ctrl.tree.getNodeList(ctrl.path),
```

Comment:
should be available as `ctrl.nodeList` already

Reply:
Oh nice; will update